Source/Windows10/Prism.Windows/PrismApplication.cs

```diff
@@ -219,9 +219,18 @@ protected virtual object Resolve(Type type)
         {
             return Activator.CreateInstance(type);
         }
-
+        /// <summary> Invoked when the application is .
```

Comment:
"Invoked when the application is ." can be removed or maybe use "Invoked when the application is activated via ShareTarget"?